src/helpers/DirectionalLightHelper.js

```diff
@@ -32,14 +34,16 @@ function DirectionalLightHelper( light, size ) {
 		- size,   size, 0
 	], 3 ) );
 
-	var material = new LineBasicMaterial( { fog: false } );
+	var material = new LineBasicMaterial( { fog: false, color: this.color } );
```

Comment:
@looeee This is a problem If `color` is  `undefined`. Multiple helpers have this issue...